TaskManager/Home/View/TaskTableViewCell.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  TaskTableViewCell.swift
+//  TaskManager
+//
+//  Created by user on 14.09.2022.
+//
+
+import UIKit
+
+class TaskTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var circleLabel: UILabel!
+    @IBOutlet weak var taskLabel: UILabel!
+    
+    
+    
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        setupCircleLabel()
+        setupTaskLabel()
+    }
+    
+    private func setupCircleLabel() {
+        circleLabel.layer.cornerRadius = circleLabel.frame.width / 2
+        circleLabel.layer.masksToBounds = true
+        let color = hexStringToUIColor(hex: "#D44F4F")
+        circleLabel.layer.borderColor = color.cgColor
+        circleLabel.layer.borderWidth = 2.0
+        
+    }
+    
+    private func setupTaskLabel() {
+        taskLabel.numberOfLines = 0
+    }
+    
+    func strikethroughText(label: UILabel) {
+        
+        let attributeString:NSMutableAttributedString = NSMutableAttributedString(string: label.text ?? "")
+        attributeString.addAttribute(NSAttributedString.Key.strikethroughStyle, value: 2, range: NSRange(location: 0, length: attributeString.length))
+        taskLabel.attributedText = attributeString
+    }
+    
+    //MARK: - Set Hex color
+    private func hexStringToUIColor (hex:String) -> UIColor {
```

Comment:
let's add this as extenstion to UIColor, so we can use it everywhere in the project. Create another file named 'UIColor+Extensions.swift' and create the extension there